README.md

```diff
@@ -12,60 +12,33 @@ Please note that this tool is currently in Preview, and need your feedback for i
-- While a container is mounted, the data in the container should not be modified by any process other than blobfuse.  This includes other instances of blobfuse, running on this or other machines.  Doing so could cause data loss or data corruption.  Mounting other containers is fine.
-- Modifications to files are not persisted to Azure Blob storage until the file is closed. If multiple handles are open to a file simultaneously, and data in the file has been modified, the close of each handle will flush the file to blob storage. 
-
-## Current Limitations
-- Some file system APIs have not been implemented: readlink, symlink, link, chmod, chown, fsync, lock and extended attribute calls.
-- Not optimized for updating an existing file. blobfuse downloads the entire file to local cache to be able to modify and update the file
-- Does not support SAS yet
-- High latency compared to local filesystems. Further you are from the Azure region, higher the latency is.
-- Currently does not implement data integrity checks. Always have your data backed up before using blobfuse. Use Https as a data integrity mechanism over the wire.
-- Because Azure Block Blobs do not have file and directory semantics, certain directory operations may not behave entirely as expected. For example, deleting the last file in a directory may also delete the directory.
-
 ## Installation
-
-You can install blobfuse from the Linux Software Repository for Microsoft products. The process is explained in the [blobfuse installation](https://github.com/Azure/azure-storage-fuse/wiki/Installation) page. Alternatively, you can clone this repository, install the dependencies (libfuse, libcurl and GnuTLS) and build from source code. See details in the [wiki](https://github.com/Azure/azure-storage-fuse/wiki/Installation#build-from-source).
+You can install blobfuse from the Linux Software Repository for Microsoft products. The process is explained in the [blobfuse installation](https://github.com/Azure/azure-storage-fuse/wiki/Installation) page. Alternatively, you can clone this repository, install the dependencies (fuse, libcurl and GnuTLS) and build from source code. See details in the [wiki](https://github.com/Azure/azure-storage-fuse/wiki/Installation#build-from-source).
 
 ## Usage
 
 ### Mounting
-Once you have built blobfuse, configure your account credentials in the template provided in blobfuse folder (connection.cfg).
+Once you have installed blobfuse, configure your account credentials either in the template provided in blobfuse folder (connection.cfg), or in the environment variables. For brevity, let's use the environment variables:
 
-Here is the format for connection.cfg:
 ```
-accountName <account-name-here> 
-accountKey <account-key-here> 
-containerName <container-name-here>
+export AZURE_STORAGE_ACCOUNT=myaccountname
+export AZURE_STORAGE_ACCESS_KEY=myaccountkey
```

Comment:
Perhaps it would be worthwhile to mention again here that they can use a sas, too, but they must have only one kind of credential specified?

Reply:
My vote is for 'Basics' in README, more options in the wiki.